### PR TITLE
P1 text utilities lists component

### PR DESCRIPTION
**Pull request by dhimanaishwarya**
Text Utilities

- headings
- small text
- gray text
- center text

Lists

- Spaced (normal lists)
- Stacked (Notification Page)

### CODE REVIEW

File: components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+.list-no-bullet > li {
+  padding: 4px 0px;
+}
+
+.checked-list {
+  list-style: none;
+}
+
+.checked-list-item {
+  margin: 8px 0px;
```
Comment (Ananyamadhu08) [inline components/list/list.css:14]: try using rem instead of px.

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go, can go ahead and merge 👍

File: components/list/list.css (hunk 1 above)
Comment (dhimanaishwarya) [reply components/list/list.css:14]: hey, I am using rem for anything above 10 px.

**Review by yashodaAgarwal — COMMENTED** [review]
Good use of variable css for color and sizing.Looks good to merge

**Pull request merged**